content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,129 @@
+
+An increasing number of systems leverage a combination of CPUs and hardware accelerators to support latency-critical execution and high-throughput parallel computation. These include workloads in fields such as telecommunications, scientific computing, machine learning, financial services and data analytics. Such hybrid systems comprise a high performance environment.
+
+In order to extract the best performance, optimizations related to CPU isolation and memory and device locality are required. However, in Kubernetes, these optimizations are handled by a disjoint set of components.
+
+_Topology Manager_ is a component in Kubelet that provides node level policies to enable these performance optimizations in an user abstract manner.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Topology Manager
+
+Currently, the CPU and Device Manager in Kubernetes make resource allocation decisions independently of each other.
+This can result in sub-optimal allocations on multiple-socketed systems, performance/latency sensitive applications
+will suffer due to these sub-optimal allocations.
+
+The Topology Manager is a new Kubelet component, that acts as a source of truth for other Kubelet components to make topology aligned resource decisions.
+
+The Topology Manager provides an interface for components, called *Hint Providers*, to send and recieve topology information. The default *best-effort* algorithm takes
+all possible NUMA Node combinations for an incoming container and chooses the best fit (being the narrowest NUMA node with resources available). The topology manager recieves
+the NUMA information from the *Hint Providers* as a bitmask and preforms a bitwise and to get the combined results, it then chooses the narrowest preffered result and stores it.
+The *Hint Providers* then use this when making the resource allocation decisions.
+
+### Configuration
+
+The Topology Manager is an alpha feature in Kubernetes v1.16 and will be disbabled by default behind a feature gate. 
+
+The Topology Manager currently:
+* Works on a nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/)
+* Works on Pods in the `Guaranteed` QOS Class {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}. 
+If these conditions are met, Topology Manager will align CPU and device requests.
+
+Topology Manager has two allocation policies, that are set via a Kubelet flag `--topology-manager-policy`.
+There are two supported policies:
+
+*`none`: the default, which means no Topology Management will take place and Kubelet will preform
+as previously.
+*`best-effort`: which means the Topology Manager will provide the best fit allocation for the container
+but will not fail the pod in the event this is not possible.
+
+### None policy

```

Comment:
```suggestion
### None policy {#policy-none}
```

This is a thing I tend to do as a kind of defensive design thing, it's not required by the style guide.
Where a fragment identifier will be linked to a keyword, I'll explicitly assign a stable name. Then, if the heading text changes, existing links keep working.